### PR TITLE
2024-03 Newsletter

### PR DESCRIPTION
**Pull request by twinkarma**

### CODE REVIEW

File: _newsletters/2024-03-14-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
+---
+layout: post
+title: March 2024 Newsletter
+editor: Twin Karmakharm
+slug: 2024-03-14-newsletter
+date: 2024-01-14T17:00:00Z+0100
```
Comment (ns-rse) [inline _newsletters/2024-03-14-newsletter.md]:
```suggestion
date: 2024-03-14T17:00:00Z+0100
```

File: _newsletters/2024-03-14-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+specific problems. These consultations can be booked via our webpage. If you are interested please visit the following
+link: <https://students.sheffield.ac.uk/it-services/research>.
+
+## Sheffield RSE Team
+
+The Sheffield RSE Team aims to [collaborate](https://rse.shef.ac.uk/collaboration/) with you to help improve your
+research software. They can [provide dedicated staff](https://rse.shef.ac.uk/collaboration/provision/) to ensure that
+you can deliver excellent research software engineering on your research projects.
+
+[RSE support via the Centre for Machine Intelligence](https://rse.shef.ac.uk/collaboration/cmi-rse/) is available to
```
Comment (ns-rse) [inline _newsletters/2024-03-14-newsletter.md]: This and the next paragraph where it talks about Code Clinics might be considered duplication of content already covered above.

**Review by ns-rse — CHANGES_REQUESTED** [review]
Thanks for putting this together @twinkarma .

I noticed the date in the header needs tweaking and some of the footer sections which is copied from previous is kind of duplication but not a big deal.

File: _newsletters/2024-03-14-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+  - Opportunity for networking.
+  - Fully funded: all tuition, materials, meals, transport to and from the Daresbury campus from Warrington, are included.
+  - Opportunity to promote posters and potentially [publish research](https://ukri.sharepoint.com/sites/thesource-stfc/SitePages/Publish-an-STFC-Report.aspx?web=1&referrer=Yammer&referrerScenario=Feed.View).
+  - Prizes for most creative approach to solving algorithms.
+  - Open badges
+- [UK Reproducibility Network Training dates for 2024]((https://www.ukrn.org/training-schedule/))
+
+---
+
+We're in the process of finalising some forthcoming RSE training events so keep an eye on our
```
Comment (tdjames1) [inline _newsletters/2024-03-14-newsletter.md]: There are several Git/GitKraken sessions lined up, could add dates here?

Comment (ns-rse) [reply _newsletters/2024-03-14-newsletter.md]: These have now been published to the RSE site and appear under the Events page linked.

I wrote them and made a PR a couple of weeks ago but forgot to merge, only realised when Joe said there was nothing listed. I started writing new pages before I remembered that the PR was waiting to be merged :facepalm: 

No harm in listing them here though to highlight they are scheduled.

File: _newsletters/2024-03-14-newsletter.md (hunk 2 above)
Comment (twinkarma) [reply _newsletters/2024-03-14-newsletter.md]: Thanks, I'll remove it.

File: _newsletters/2024-03-14-newsletter.md (hunk 3 above)
Comment (twinkarma) [reply _newsletters/2024-03-14-newsletter.md]: Cheers @tdjames1 I'll list the next few git session here

**Review by ns-rse — APPROVED** [review]
Cool, good work @twinkarma and thanks for adding the Git courses and other training details.

**Comment by twinkarma** [comment]
Thanks for your reviews @ns-rse and @tdjames1 !

**Pull request merged**